### ISSUE TITLE
Overall todo list for spec doc milestone

### ISSUE THREAD

**Issue by lwjohnst86** [steno-aarhus/seedcase-project#53]
This issue will track some todo items related to completing a specification report that we can send to get externally reviewed. The main issue tracker related to it though is this [milestone](https://github.com/steno-aarhus/seedcase-project/milestone/1). I'll add todo items as we progress. Ultimately, we want to have a final spec doc that I've created at [`design/_report.qmd`](design/_report.qmd).

Inside the report document are "tags" that look like `{{ include FILENAME }}`. These include tags will insert the contents of the file into that location in the document. So each of those files need to be completed before this report is ready to be sent out for review. I'll assign each file to a person for them to complete and write *inside that file*. I'll list all the files to complete below (with the assigned person). Issues specific to each file that come up, I will create a new issue and put it into the milestone.

File and assignment list will be added here shortly.

**Comment by lwjohnst86** [comment]
These are some links (mainly for me) that describe how to write architecture documents:

- https://c4model.com/
- https://www.linkedin.com/pulse/effective-architecture-documentation-arc42-c4-torsten-mosis/
- https://www.innoq.com/en/blog/brief-introduction-to-arc42/#fn:1
- https://www.edrawsoft.com/article/data-architecture-diagram.html
- https://arc42.org/
- https://thepracticaldeveloper.com/
- https://nocomplexity.com/documents/arplaybook/introduction.html

Examples:

- https://biking.michael-simons.eu/docs/index.html#_stakeholders
- https://hsc.aim42.org/documentation/hsc_arc42.html#quality-goals
-
